parser/parser.go

```diff
@@ -5,13 +5,16 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
+	"path/filepath"
 	"regexp"
 
 	"github.com/dogtools/dog/types"
 	"github.com/ghodss/yaml"
 )
 
 var ErrMalformedStringArray = errors.New("Malformed strings array")
+var ErrNoDogfiles = errors.New("No dogfiles found")
```

Comment:
I'd say "No Dogfile found" here, to keep it consistent with the documentation.

Note: At some point we'll need a way to specify (in errors, in the docs, etc) if we are talking about the file itself or the fact that a folder have 1 multiple Dogfiles that together form a "Dogfile" as defined in the Spec. Both things have the same name now and it's super confusing.